src/Components/Panels/Toolbox/BaseControl.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { IBase } from '../../../Document/Obstacles/Base';
+
+import styles from './BaseControl.module.scss';
+
+interface Props {
+  data: IBase;
+  onChange: (changes: IBase) => void;
+}
+
+const BaseControl = ({ data, onChange }: Props) => {
+  const handleBaseColourOnChange = (value: number) => {
+    onChange({
+      ...data,
+      color: value as IBase['color'],
+    });
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <fieldset className={styles.base}>
+        <legend className={styles.legend}>Base</legend>
+        <div>
+          <select
+            className={styles.dropdown}
+            id="base-color"
+            value={data.color}
+            onChange={(event) =>
+              handleBaseColourOnChange(parseInt(event.target.value))
+            }
```

Comment:
```suggestion
            onChange={handleBaseColorOnChange}
```

If you can [avoid arrow functions as a prop](https://stackoverflow.com/a/36677798/1239484), it's preferable to do so.